addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -150,20 +150,37 @@ var KanbanView = BasicView.extend({
                 defaultValues[match[1]] = self.loadParams.context[key];
             }
         });
+        var searchPanel;
```

Comment:
why is it declared here?

Reply:
also, we are in 2020 :D

Reply:
Not in 12.3, sadly :(

Reply:
> why is it declared here?

Because of scoping: this variable is used in both `then` blocks

Reply:
> Not in 12.3, sadly :(

ho I thought 13.3 sorry

Reply:
ok

Reply:
> Not in 12.3, sadly :(

ho I thought 13.3 sorry